.istanbul.yml

```diff
@@ -0,0 +1,6 @@
+instrumentation:
+    excludes:
+        - spec/**/*.js
+        - lib/index.js
+        - bin/manual-testing/*
```

Comment:
Good catch. This line has been removed, as those manual scripts will be moved into DDQ.